delete_dashboards_by_text_in_title/delete_dashboards.py

```diff
@@ -0,0 +1,45 @@
+from datadog import initialize, api
+
+
+def delete_dashboards_by_text_in_title():
+
+    api_key = input("Please enter your API key: ")
+    app_key = input("Please enter your APPLICATION key: ")
+
+    options = {"api_key": api_key, "app_key": app_key}
+
+    initialize(**options)
+
+    text_to_search_in_dashboard = input(
```

Comment:
I would do some validation around this to make sure there is some text and that text is not space or something that is going to return many unintentional dashboards.